simpledotorg/simple-server
# Dashboard: Add missing table key value (>90 patient days of stock)

**Story card:** none

## Because

There is a missing key value '>90 patient days of stock'

![Screenshot 2022-09-22 at 14 23 03](https://user-images.githubusercontent.com/9541902/191758803-b5ceb966-74e0-448c-90bf-d90d73ca7328.png)

## This addresses

Green colour showing on the table data but no key to explain why.

![Screenshot 2022-09-22 at 14 20 02](https://user-images.githubusercontent.com/9541902/191758920-9e0f8225-20c4-4fe7-82c4-e52e13dc9bc4.png)

## Test instructions

![Screenshot 2022-09-22 at 14 22 29](https://user-images.githubusercontent.com/9541902/191758847-dfd65677-de62-422d-8c44-ba879b584fcf.png)

Found at: http://localhost:3000/my_facilities/drug_stocks?facility_group=aligarh&for_end_of_month=Aug-2022&size=small

Scroll to page bottom.

To check colour match insert data into CCB Tablets (this works) using the '+' button to add drug data.
